config/src/config/logger_config.rs

```diff
@@ -10,13 +11,35 @@ pub struct LoggerConfig {
     pub is_async: bool,
     // chan_size of slog async drain for node logging.
     pub chan_size: usize,
+    // The logging level for Slog logging.
+    // Due to FilterLevel not supporting serde, we
+    // represent the level as a String.
+    pub filter_level: String,
```

Comment:
We should make an enum of levels to handle this and then implement an into or from for these types.

Reply:
FilterLevel itself is an enum (https://docs.rs/slog/2.0.0-3.1/slog/enum.FilterLevel.html), and its implementation provides appropriate to_str() and from_str() methods. As such, I think it's cleaner to use those directly for serialization/deserialization, as: (i) it avoids me having to re-implement another layer of indirection using an additional enum; and (ii) the string output in the config is human readable (in case someone wishes to manually modify the config themselves).